node-src/lib/getConfiguration.ts

```diff
@@ -48,8 +49,20 @@ const configurationSchema = z
 
 export type Configuration = z.infer<typeof configurationSchema>;
 
+function resolveConfigFileName(configFile?: string): string {
+  const usedConfigFile = [
+    configFile,
+    'chromatic.config.json',
+    'chromatic.config.jsonc',
+    'chromatic.config.json5',
+  ].find((f?: string) => f && existsSync(f));
+
+  return usedConfigFile || 'chromatic.config.json';
+}
 /**
- * Parse configuration details from a local config file (typically chromatic.config.json).
+ * Parse configuration details from a local config file (typically chromatic.config.json, but can
+ * also use the JSON5 .jsonc extension. If both files are present, then the .json will take
+ * precedence.
```

Comment:
Should this update with all the new supported extensions?

Reply:
Yup, I'll fix.